app/ts/Subproblem/ScaleRanges/ScaleRangesUtil.ts

```diff
@@ -18,6 +21,11 @@ export function getScaleRangeWarnings(workspace: IWorkspace): string[] {
       'Criterion with only missing or text values selected, therefore no scales can be set.'
     );
   }
+  if (hasRowWithOnlySameValue(observedRanges)) {
+    warnings.push(
+      'Criterion with the same value set for all alternatives selected, therefore no scales can be set.'
```

Comment:
Criterion selected where all values are identical, therefore no scales can be set.